charts/vault-monitoring/templates/servicemonitor.yaml

```diff
@@ -24,9 +24,11 @@ spec:
       port: {{ .Values.vault.portName }}
       scheme: {{ .Values.vault.scheme }}
       scrapeTimeout: {{ .Values.vault.serviceMonitor.scrapeTimeout }}
+    {{- if eq .Values.vault.serviceMonitor.authentication true }}
       {{- with .Values.vault.serviceMonitor.bearerTokenFile }}
       bearerTokenFile: {{ . | quote }}
```

Comment:
We should revert @pree commit here. No need to check bearetoken is set if authentication is true

Reply:
Reverted the change in the next commit